app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListFragment.kt

```diff
@@ -90,6 +92,17 @@ class OwnerListFragment : BaseViewBindingFragment<FragmentOwnerListBinding>(), O
     override fun onResume() {
         super.onResume()
         viewModel.loadOwners()
+
+        //FIXME: find better way to pass results in nav graph
```

Comment:
This StackOverflow question might have an answer for us. I am not sure which one really works and is better than what we have today: https://stackoverflow.com/questions/50334550/navigation-architecture-component-passing-argument-data-to-the-startdestination

Reply:
I would create separate issue for this refactoring (and maybe investigating other available alternatives) as it touches places not related to this feature.